src/assets/stylesheets/layout/_topics.scss

```diff
@@ -9,18 +9,14 @@
     border: 1px solid #6695FF;
     padding: 1rem;
     border-radius: 0.25rem;
-    flex-basis: 33%;
-    margin: 0 1rem;
```

Comment:
The property `flex-basis` set to a value smaller than 33%, together with `justify-content: space-between;` ensures that there's a small space between the cards. And the `margin` isn't needed anymore. :raised_hands: